app/react/Templates/components/ViewTemplateAsPage.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect } from 'react';
```

Comment:
ViewTemplateAsPage reads like this is a view of some sort, can we provide a better name that hints that this is a Control of some sort ?

Reply:
Yes.  Good idea.